hecuba_py/tests/withcassandra/storageobj_tests.py

```diff
@@ -1236,6 +1240,13 @@ def test_nested_recreation(self):
         self.assertEqual(external_sobj.myotherso.name, name_attr)
         self.assertEqual(external_sobj.myotherso.age, age_attr)
 
+    def test_timestamp(self):
+        config.session.execute("DROP TABLE IF EXISTS my_app.timestampAttrib")
+        d = TestTimestamp("my_app.timestampAttrib")
+        timestamp = 1545733000
+        dt = datetime.fromtimestamp(timestamp)
```

Comment:
This test was only working because the StorageObj was returning the in-memory value